warehouse/ingest-configuration/src/main/resources/config/log4j-flagmaker.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+<log4j:configuration xmlns:log4j="http://jakarta.apache.org/log4j/">
+
+    <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
+    	<param name="Target" value="System.out" />
+		<layout class="org.apache.log4j.PatternLayout">
+			<param name="ConversionPattern" value="%d{ISO8601} %p [%c] %m%n"/>
+		</layout>
+    </appender>
+
+	<logger name="org.apache.hadoop">
+		<level value="info"/>
+	</logger>
+	
+	<root>
+		<priority value="debug"/>
+		<appender-ref ref="CONSOLE"/>
+	</root>
+</log4j:configuration>
```

Comment:
The spacing in this file looks pretty inconsistent. Maybe run it through a formatter?